prefect_email/credentials.py

```diff
@@ -79,8 +80,11 @@ class EmailCredentials:
         smtp_port: If provided, overrides the smtp_type's default port number.
     """
 
+    _block_type_name = "Email Credentials"
+    _logo_url = "https://github.com/PrefectHQ/orion/blob/main/docs/img/collections/email.png?raw=true"  # noqa
```

Comment:
I uploaded the image to our CMS.
```suggestion
    _block_type_name = "Email Server Credentials"
    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/3PcxFuO9XUqs7wU9MiUBMg/ca740e27815d15528373aced667f58b9/email__1_.png?h=250"  # noqa
```